packages/keycodes/src/index.ts

```diff
@@ -157,29 +178,30 @@ function capitaliseFirstCharacter( string ) {
 /**
  * Map the values of an object with a specified callback and return the result object.
  *
- * @template {{ [s: string]: any; } | ArrayLike<any>} T
- *
- * @param {T}                     object Object to map values of.
- * @param {( value: any ) => any} mapFn  Mapping function
+ * @template T The object type
+ * @template R The return type of the mapping function
  *
- * @return {any} Active modifier constants.
- */
-function mapValues( object, mapFn ) {
+ * @param    object - Object to map values of.
+ * @param    mapFn  - Mapping function to apply to each value.
```

Comment:
The formatting on these parameter descriptions is off:

```suggestion
 * @param    object Object to map values of.
 * @param    mapFn  Mapping function to apply to each value.
```